exporter/containerimage/writer.go

```diff
@@ -419,11 +419,14 @@ func (ic *ImageWriter) rewriteRemoteWithEpoch(ctx context.Context, opts *ImageCo
 	var divergedFromBase bool
 	for i, desc := range remoteDescriptors {
 		i, desc := i, desc
-		info, err := cs.Info(ctx, desc.Digest)
-		if err != nil {
-			return nil, err
+		diffID := digest.Digest(desc.Annotations[labels.LabelUncompressed]) // can be empty
+		if diffID == "" {
+			info, err := cs.Info(ctx, desc.Digest)
+			if err != nil {
+				return nil, err
+			}
+			diffID = digest.Digest(info.Labels[labels.LabelUncompressed]) // can be still empty
```

Comment:
Can you update these comments a bit. In what case/why can they be empty and why is it ok that they are empty.

Reply:
done